audio/song_list.inc.c

```diff
@@ -0,0 +1,1930 @@
+// Size of the Sound List (-1)
+u32 D_08a9e8e0 = 1924 - 1;
+
+// Unused List of Sounds
+struct SongHeader *D_08a9e8e4[] = {
+    /* 0000 */ NULL,
```

Comment:
mentioned this in discord but this and song_table.inc.c should be done with a cross macro so that the repeated data is only present in the repo once

Reply:
I've managed to do this, but I'm unsure of how exactly you wanted it to look/function. Feel free to point this in a more specific direction.

Reply:
I've updated this once again, following the directions provided on Discord.